### DISCUSSION TITLE
How to assign multiple 'Temperaure Sources' for a graph?

### DISCUSSION THREAD

**Discussion by miguel1900** [Rem0o/FanControl.Releases#331] [Q&A] [answered]
Hi,

I was wondering if it possible to set multiple temperature sources for an unique fan curve. I simply want to adapt the RPM of a fan depending on the CPU temperature but also the GPU or even MOBO temperatures.

I was loving this app and I was switching to use and recommend it always, until I discovered this limitation.

Thank you very much, best regards, and congrats for the great work!!

**Comment by Rem0o** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/14118956/132953742-a4ab54d9-5856-492d-b6d7-9183c59cbda9.png)

Then use that sensor for your graph.

**Reply by miguel1900** [reply]
Wow, that's so incredible!

Thanks and congrats again!

**Comment by whoozy82** [comment]
I was about to request that exact feature and then realized, that it's already there.
And the combined fan curves are even better   :-)

Great app! I love it.
